### ISSUE TITLE
Metric for evaluating <extract_data_table> task

### ISSUE THREAD

**Issue by nourheshamshaheen** [vis-nlp/UniChart#6]
Hello! 
What is the metric used to finetune the model for the <extract_data_table> task?

**Comment by AhmedMasryKU** [comment]
Hello!
Thanks for your interest in our work. 
We will release the metrics, datasets, and the code for the Chart-to-Table task this week.

**Comment by ChangGuiyong** [comment]
Hi, how to fine tune the <extract_data_table> task, can the model support Chinese well?

**Comment by whishei** [comment]
Hi!
I am interested in fine-tuning your model for the the Chart-to-Table task. Will you be releasing the code for the Chart-to-Table task?
